### ISSUE TITLE
propmting textinput value not committed on change

### ISSUE THREAD

**Issue by nicoulaj** [flex-users/flexlib#43]
Originally filed by squarefa...@gmail.com on 2007-07-11T18:23:05

What steps will reproduce the problem?
1. Create a binding function that passed in the text of a prompting input
2. trace the text when it fires
3. Change the text in the prompting input

What is the expected output? What do you see instead?
I expected the binding to immediately get the new text, I didn't get the new text until I clicked on 
and off of another component.

What version of the product are you using? On what operating system?
1.9, osx

Please provide any additional information below.

**Comment by nicoulaj** [comment]
Updated by dmcc...@gmail.com on 2007-09-09T00:43:47

I don't get the same result if I use ChangeWatcher.watch() to watch the text property
of a PromptingTextInput. I get the correct value reported on each change event. 

Tested on flexlib version .1.9 on OSX.

I'm marking this as invalid. If someone can reproduce we will re-open this issue.

Original ticket set status to Invalid (we converted to closed)
